Cargo.toml

```
@@ -16,3 +16,6 @@ members = [
 
 [profile.dev]
 opt-level = 2
+
+[patch.crates-io]
+elliptic-curve = { git = "https://github.com/ycscaly/traits", branch = "batch_invert" }
```

Now that the `branch_invert` branch is merged:

```suggestion
elliptic-curve = { git = "https://github.com/RustCrypto/traits" }
```